### PR TITLE
Reformat

### PR DESCRIPTION
**Pull request by ghost**
Indents and Spaces Fixed

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -310,11 +349,11 @@ p {
...
     text-overflow: clip;
     /*background: rgba(1, 1, 2, 0.47);*/
-    -webkit-transform:    skewX(-20deg);
-    transform:    skewX(-20deg);
+    -webkit-transform: skewX(-20deg);
+    transform: skewX(-20deg);
 }
 
-.parallelogram > a > span {
+.parallelogram>a>span {
```
Comment (dsouzaedison) [inline styles.css]: Provide proper spacing.

File: styles.css [hunk 2]
```diff
@@ -424,7 +463,7 @@ p {
     transition: all 0.5s ease;
 }
 
-.home-gallery-image > div {
+.home-gallery-image>div {
```
Comment (dsouzaedison) [inline styles.css]: Spacing

File: styles.css [hunk 3]
```diff
@@ -511,7 +554,7 @@ p {
     vertical-align: middle;
 }
 
-.service-icon > i {
+.service-icon>i {
```
Comment (dsouzaedison) [inline styles.css]: Spacing

File: index.html [hunk 4]
```diff
@@ -2,21 +2,21 @@
...
 
 <head>
-    <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
     <title>Karnataka Tours & Services</title>
-    <meta name="author" content="Matthew Howell"/>
-    <meta name="description" content="fullPage continuous scrolling demo."/>
-    <meta name="keywords" content="fullpage,jquery,demo,scroll,loop,continuous"/>
-    <meta name="Resource-type" content="Document"/>
+    <meta name="author" content="Matthew Howell" />
```
Comment (dsouzaedison) [inline index.html]: Remove unwanted meta tags

File: index.html [hunk 5]
```diff
@@ -45,239 +45,242 @@
...
             </div>
         </div>
-    </div>
 
-    <!--<li data-menuanchor="firstPage" class="active"><a href="#firstPage">First slide</a></li>-->
-    <!--<li data-menuanchor="secondPage"><a href="#secondPage">Second slide</a></li>-->
-    <!--<li data-menuanchor="3rdPage"><a href="#3rdPage">Third slide</a></li>-->
-    <!--<li data-menuanchor="4thpage"><a href="#4thpage">Fourth slide</a></li>-->
-</ul>
+        <!--<li data-menuanchor="firstPage" class="active"><a href="#firstPage">First slide</a></li>-->
```
Comment (dsouzaedison) [inline index.html]: Remove this if not needed

**Review by dsouzaedison — CHANGES_REQUESTED** [review]

**Comment by dsouzaedison** [comment]
Correct all spacing issues in this PR itself.

File: index.html [hunk 6]
```diff
@@ -2,31 +2,12 @@
 <html xmlns="http://www.w3.org/1999/xhtml">
 
 <head>
-    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
```
Comment (dsouzaedison) [inline index.html]: This tag is necessary

Comment (Parth-Agarwal) [reply index.html]: <meta charset="UTF-8"> is used in html5. The one above is html 4.0.1

Comment (dsouzaedison) [reply index.html]: Have you added this `<meta charset="UTF-8">` in the index file?

**Review by dsouzaedison — APPROVED** [review]

**Pull request merged**